Runtime code generation issue with Orleankka, TestCluster and OrleansCodeGenerator 2.2.3

When using `Microsoft.Orleans.OrleansCodeGenerator` 2.2.3 with `Orleankka.FSharp` 2.2, the runtime code generation fails for `Orleans.TestingHost.TestCluster` with the following error (taken from the reproduction below):
```
 Orleans.CodeGenerator.CodeGenerationException : (1,7602): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
(1,21309): error CS0432: Alias 'global' not found
(1,21357): error CS0432: Alias 'global' not found
(1,21406): error CS0432: Alias 'global' not found
(1,21793): error CS0432: Alias 'global' not found
(1,21841): error CS0432: Alias 'global' not found
(1,21890): error CS0432: Alias 'global' not found
(1,22366): error CS0432: Alias 'global' not found
(1,22414): error CS0432: Alias 'global' not found
(1,22463): error CS0432: Alias 'global' not found
(1,25753): error CS0432: Alias 'global' not found
(1,25801): error CS0432: Alias 'global' not found
(1,25850): error CS0432: Alias 'global' not found
(1,26254): error CS0432: Alias 'global' not found
(1,26302): error CS0432: Alias 'global' not found
(1,26351): error CS0432: Alias 'global' not found
(1,9438): error CS0432: Alias 'global' not found
(1,9486): error CS0432: Alias 'global' not found
(1,9535): error CS0432: Alias 'global' not found
(1,9937): error CS0432: Alias 'global' not found
(1,9985): error CS0432: Alias 'global' not found
(1,10034): error CS0432: Alias 'global' not found
(1,12327): error CS0432: Alias 'global' not found
(1,12375): error CS0432: Alias 'global' not found
(1,12424): error CS0432: Alias 'global' not found
(1,14807): error CS0432: Alias 'global' not found
(1,14855): error CS0432: Alias 'global' not found
(1,14904): error CS0432: Alias 'global' not found
(1,15174): error CS0432: Alias 'global' not found
(1,15222): error CS0432: Alias 'global' not found
(1,15271): error CS0432: Alias 'global' not found
Stack Trace:
   at Orleans.CodeGenerator.RoslynCodeGenerator.CompileAssembly(GeneratedSyntax generatedSyntax, String assemblyName, Boolean emitDebugSymbols)
   at Orleans.CodeGenerator.RoslynCodeGenerator.GenerateAndLoadForAssemblies(IEnumerable`1 assemblies)
   at Orleans.Hosting.ApplicationPartManagerCodeGenExtensions.WithCodeGeneration(IApplicationPartManagerWithAssemblies manager, ILoggerFactory loggerFactory)
   at Orleankka.Cluster.SiloHostBuilderExtension.<>c.<UseOrleankka>b__2_1(IApplicationPartManager apm) in C:\Work\OSS\Orleankka\Source\Orleankka.Runtime\Cluster\ClusterOptions.cs:line 143
   at Orleans.Hosting.SiloHostBuilderExtensions.ConfigureApplicationParts(ISiloHostBuilder builder, Action`1 configure)
   at Program.SiloConfigurator.Orleans-TestingHost-ISiloBuilderConfigurator-Configure(ISiloHostBuilder siloBuilder) in C:\Projects\orleans\Samples\2.1\FSharp.NetCore\test\FSharp.NetCore.GrainTests\Program.fs:line 30
   at Orleans.TestingHost.TestClusterHostFactory.ConfigureAppServices(IConfiguration configuration, ISiloHostBuilder hostBuilder)
   at Orleans.TestingHost.TestClusterHostFactory.CreateSiloHost(String hostName, IEnumerable`1 configurationSources)
   at Orleans.TestingHost.InProcessSiloHandle.CreateAsync(String siloName, IList`1 configurationSources)
   at Orleans.TestingHost.TestCluster.StartSiloAsync(Int32 instanceNumber, TestClusterOptions clusterOptions, IReadOnlyList`1 configurationOverrides, Boolean startSiloOnNewPort)
   at Orleans.TestingHost.TestCluster.InitializeAsync()
   at Orleans.TestingHost.TestCluster.DeployAsync()
   at Orleans.TestingHost.TestCluster.DeployAsync()
   at Orleans.TestingHost.TestCluster.Deploy()
   at Program.Tests..ctor() in C:\Projects\orleans\Samples\2.1\FSharp.NetCore\test\FSharp.NetCore.GrainTests\Program.fs:line 41
```
So far I was only able to reproduce it with the test cluster and using Orleankka. Even with Orleankka the production silo and client work fine with 2.2.3 code generator. I'm not sure if F# is also a factor, I haven't tried with C#.

I have reproduction as a unit test in the F# .NET Core sample:
https://github.com/jbrestan/orleans/tree/codegen-runtime-repro/Samples/2.1/FSharp.NetCore/test/FSharp.NetCore.GrainTests
Running `dotnet test` should result in the error message above. If you downgrade OrleansCodeGenerator to 2.2, the test should pass as expected.

Let me know if I can provide more details.

--- a/src/Orleans.CodeGeneration/Utilities/SyntaxFactoryExtensions.cs
+++ b/src/Orleans.CodeGeneration/Utilities/SyntaxFactoryExtensions.cs
@@ -111,8 +111,8 @@ namespace Orleans.CodeGenerator.Utilities
                     "bindingFlags", 
                     string.Format("Can't create parenthesized binary expression with {0} arguments", bindingFlags.Length));
             }
-
-            var flags = SyntaxFactory.AliasQualifiedName("global", SyntaxFactory.IdentifierName("System")).Member("Reflection").Member("BindingFlags");
+            
+            var flags = SyntaxFactory.AliasQualifiedName(SyntaxFactory.IdentifierName(SyntaxFactory.Token(SyntaxKind.GlobalKeyword)), SyntaxFactory.IdentifierName("System")).Member("Reflection").Member("BindingFlags");
             var bindingFlagsBinaryExpression = SyntaxFactory.BinaryExpression(
                 operationKind,
                 flags.Member(bindingFlags[0].ToString()),

